subsys/bluetooth/audio/bap_broadcast_source.c

```diff
@@ -885,6 +961,24 @@ int bt_bap_broadcast_source_reconfig(struct bt_bap_broadcast_source *source,
 
 			stream_param = &subgroup_param->params[j];
 			stream = stream_param->stream;
+			if (CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE > 0) {
+				if (bis_count >= BROADCAST_STREAM_CNT) {
+					LOG_DBG("Stream count %d exceeded", bis_count);
+					return -ENOMEM;
+				}
```

Comment:
We are not adding new BIS in this function. Suggest to remove all uses of `bis_count` in this function